Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -2976,14 +2976,12 @@ Element* AccessibilityObject::element() const
 
 const RenderStyle* AccessibilityObject::style() const
 {
-    const RenderStyle* style = nullptr;
     if (auto* renderer = this->renderer())
-        style = &renderer->style();
-    if (!style) {
-        if (auto* element = this->element())
-            style = element->computedStyle();
-    }
-    return style;
+        return &renderer->style();
+
+    if (auto* element = this->element())
+        return element->computedStyle();
+    return nullptr;
```

Comment:
This is a change in behavior. Before, if `renderer` was non-null, but `renderer->style()` was null, we would fallback to getting style from the element. Now we don't do that.

That being said, if we don't have a test showing why this would be important, maybe we make this change to simplify and wait for a motivating case to actually do this more complex thing?

Reply:
RenderObject::style returns a RenderStyle& which has a non-null address, so there is no change in behavior here.

Reply:
Ah right! Good to go then.